packages/pwa-kit-runtime/src/utils/ssr-config.server.test.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-env jest */
+/* eslint max-nested-callbacks:0 */
+
+import {getConfig} from './ssr-config.server'
+
+describe('Server "getConfig"', () => {
+    const env = process.env
+
+    beforeEach(() => {
+        jest.resetModules()
+        process.env = {...env}
+    })
+
+    afterEach(() => {
+        process.env = env
```

Comment:
Minor: Should this be the same as on line 18 above?

Reply:
I essentially borrow that from [here](https://www.webtips.dev/how-to-mock-processenv-in-jest). I believe the idea is to restore to the actual value and not a copy of the original values. And to also only modify a copy, not modify (add/remove) values during the rest run.